gpytorch/kernels/gpfa_kernel.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
To me, this kernel seems really similar to the [LCM Kernel](https://github.com/cornellius-gp/gpytorch/blob/master/gpytorch/kernels/lcm_kernel.py) that we already have in GPyTorch. Is there any way we can integrate the two?

I know that you want to be able to do posterior inference on the latent function (currently not totally possible with GPyTorch), and I'm not sure if this is possible with the current LCMKernel implementation. However, I'm afraid that having two similar kernels in the library will be difficult and confusing to maintain.